.changeset/nervous-suns-prove.md

```diff
@@ -0,0 +1,19 @@
+---
+"@vue-storefront/sdk": minor
+---
+
+[ADDED] `logger` option to the `middlewareModule` config. This option allows you to turn on/off the logging of the SDK requests and responses or to provide a custom logger function.
+
+```diff
+import { initSDK, buildModule, middlewareModule } from "@vue-storefront/sdk";
+import { Endpoints } from "@vsf-enterprise/sapcc-api";
+
+const sdk = initSDK({
+  commerce: buildModule(middlewareModule<Endpoints>, {
+    apiUrl: "http://localhost:8181/commerce",
++   logger: true,
```

Comment:
We should enable it by default in unified, no?

Reply:
I'm not sure, the amount of logs will be huge, and it may increase the cost for the infra. I think it may be better to set the environment variable on demand in the console to turn on the logging

Reply:
true, makes sense!